metric/unit/unit.go

```diff
@@ -16,6 +16,7 @@ package unit // import "go.opentelemetry.io/otel/metric/unit"
 
 type Unit string
 
+//nolint:revive
```

Comment:
I would like to know what was ignored and why
```suggestion
//nolint:revive // ignoring missing comments for obvious constants
```